### PR TITLE
Run clang-format

### PR DESCRIPTION
**Pull request by drwells**
It looks like we didn't run clang-format for NWT or when I updated the copyright notices.

Part of #749.

### CODE REVIEW

File: src/IB/BrinkmanPenalizationRigidBodyDynamics.cpp [hunk 1]
```diff
@@ -360,7 +361,8 @@ BrinkmanPenalizationRigidBodyDynamics::demarcateBrinkmanZone(int u_idx, double t
     // Get the interpolated density variable
     const int rho_ins_idx = d_fluid_solver->getLinearOperatorRhoPatchDataIndex();
 
-    // Get the solid level set data. We have already copied new data into scratch data and have also
+    // Get the solid level set data. We have already copied new data into scratch
+    // data and have also
     // filled its ghost cells on the finest level.
```
Comment (abarret) [inline src/IB/BrinkmanPenalizationRigidBodyDynamics.cpp]: This line should be merged with above line.

File: ibtk/src/lagrangian/LEInteractor.cpp [hunk 2]
```diff
@@ -1019,8 +1019,8 @@ perform_mls(const int stencil_sz,
                 const hier::Index<NDIM> idx(ic0, ic1);
                 T[i1][i0] = D[0][i0] * D[1][i1] * mask_data(idx, /*depth*/ 0);
 #elif (NDIM == 3)
-                const hier::Index<NDIM> idx(ic0, ic1, ic2);
```
[the remark this answers is not in the document]
Comment (drwells) [reply ibtk/src/lagrangian/LEInteractor.cpp]: I think this depends a lot on which version of clang-format is used. I'll try this with a newer one.

[the remark this answers is not in the document]
Comment (drwells) [reply ibtk/src/lagrangian/LEInteractor.cpp]: I get the same result with my newer version too. I'll undo this change manually for now.

**Comment by drwells** [comment]
Hm - it looks like 6.0 and 9.0 both mess up the indentation of those blocks. I fixed it manually. I also reflowed a bunch of comment paragraphs.

We should bless exactly one version of clang-format to avoid these problems.

**Comment by boyceg** [comment]
> We should bless exactly one version of clang-format to avoid these problems.

You mean the version that I have on my laptop? :-D

It really is too bad that a formatting tool does not have a way to enforce, ummm, standard formatting between different versions.

**Comment by drwells** [comment]
> You mean the version that I have on my laptop? :-D

works for me!


> It really is too bad that a formatting tool does not have a way to enforce, ummm, standard formatting between different versions.

Yeah.

**Pull request merged**